portal/common/lib/routing.ts

```diff
@@ -89,11 +87,16 @@ function parseRoutesData(bcsBytes: string): Routes {
  * @param path - The path to match.
  * @param routes - The routes to match against.
  */
-export function matchPathToRoute(path: string, routes: Routes) {
+export function matchPathToRoute(path: string, routes: Routes): string | undefined {
+    if (routes.routes_list.size == 0) {
+        // If the map is empty there is no match.
+        return undefined;
+    }
+
     // TODO: improve this using radix trees.
-    const res = Array
-        .from(routes.routes_list.entries())
-        .filter(([pattern, _]) => new RegExp(`^${pattern.replace('*', '.*')}$`).test(path))
-        .reduce((a, b) => a[0].length >= b[0].length ? a : b)
+    const res = Array.from(routes.routes_list.entries())
+        .filter(([pattern, _]) => new RegExp(`^${pattern.replace("*", ".*")}$`).test(path))
+        .reduce((a, b) => (a[0].length >= b[0].length ? a : b));
```

Comment:
Nit: Was this done automatically by the IDE? I would say the first indentation was more readable.

Reply:
yep, I think this is still the `pre-commit` hook.